ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -0,0 +1,33 @@
+binary_build_commands: make build
```

Comment:
@petr-muller  Seems like this `make build` didn't run after checking the [failed Prow job log](https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9282/rehearse-9282-pull-ci-openshift-openshift-tests-private-master-e2e-cmd/5), or is anything I missed? Thanks!
```golang
Updated property [core/project].
/go/src/github.com/openshift/openshift-tests-private
/bin/bash: line 77: ./bin/extended-platform-tests: No such file or directory
---
```

Reply:
The template jobs using the `openshift_installer_src` always run in a container that only contains cloned source code, not the `bin` image. Whatever you run in `openshift_installer_src` only has the source: if it needs binaries it needs to build them itself.

I think, given this is a CI config for a new repository, you should consider using multi-stage mechanism for setting up e2e test instead of templates (which are considered a legacy, will get no new features and have a number of quirks and inconsistencies, like the one you encountered here). You can find the docs for multi-stage [here](https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help), especially the examples in [this section](https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help#config).

For example, one of your current tests could be expressed like this:

```yaml
tests:
- as: e2e-steps
  steps:
    cluster_profile: aws
    workflow: ipi-aws
    test: 
    - as: e2e-test
      commands: ./bin/extended-platform-tests run openshift/conformance/serial --run "\[Serial\]
    olm version should contain the source commit id"
      from: bin
      resources:
        requests:
          cpu: 100m
          memory: 200Mi
```

Reply:
@petr-muller Many thanks for your help!  I will have a try.